connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -160,18 +156,12 @@ public void testStartStop() throws Exception {
         assertEquals(CONSUMER_ASSIGNMENT, consumer.assignment());
 
         store.stop();
-
-        assertFalse(Whitebox.<Thread>getInternalState(store, "thread").isAlive());
```

Comment:
Aren't we losing this test coverage? Can we bump up the visibility of the `thread` instance variable to `package-private` (with a comment stating that it is being made visible for testing) and retain this coverage?